src/vrm/springbone/VRMSpringBoneImporter.ts

```diff
@@ -149,7 +149,7 @@ export class VRMSpringBoneImporter {
           return;
         }
 
-        const offset = _v3A.set(collider.offset.x, collider.offset.y, collider.offset.z);
+        const offset = _v3A.set(collider.offset.x, collider.offset.y, -collider.offset.z);
```

Comment:
xもzと同様に-xする必要がありませんか？
```suggestion
        const offset = _v3A.set(-collider.offset.x, collider.offset.y, -collider.offset.z);
```

Reply:
口頭で説明した